### ISSUE TITLE
Accessibility: MAS17: Color Contrast : Luminosity contrast ratio for extension version text does not match 4.5:1

### ISSUE THREAD

**Issue by greazer** [Microsoft/vscode#29349]
Platform:
Microsoft Windows RS1
Version:10.0.14393.821
Microsoft Visual Studio Enterprise 2017 RC
Visual Studio/15.0.0-RC.4+26206.0

Steps to reproduce:
1. Launch Visual Studio Code.
2.Navigate to view menu.
3. Select "Extension" from view menu.
4. Observe the color contrast ratio for the Extension Version using contrast Analyzer.

Expected Result:
Luminosity contrast ratio for the text should be 4.5:1.

Actual Result:
Luminosity contrast ratio for the text is 4.3:1.

User Impact:
People with low vision often have difficulty while reading the selected text that do not contrast with their background color.

![color contrast ratio for the extension version is minimum](https://user-images.githubusercontent.com/11685408/27495506-fda89130-5806-11e7-9ae1-c9479e3e8cab.png)
